xonsh/tools.py

```diff
@@ -521,19 +513,16 @@ def always_false(x):
 
 def ensure_string(x):
     """Returns a string if x is not a string, and x if it already is."""
-    if isinstance(x, string_types):
-        return x
-    else:
-        return str(x)
+    return str(x)
```

Comment:
Interesting. I can't find anything in the Python docs that say this is guaranteed. Do you know of any?